src/models/default_configs.rs

```diff
@@ -25,15 +25,15 @@ pub const JAVA_CS: &str = "java_cs";
 pub const KOTLIN: &str = "kt";
 pub const KOTLIN_ALIAS: &str = "kotlin";
```

Comment:
Continuing the discussion above, here we have an alias for kotlin, can we look at the uses of this and see if we need `PYTHON_ALIAS` and `TYPESCRIPT_ALIAS`?